notifications_utils/template.py

```diff
@@ -166,12 +166,12 @@ def prefix(self, value):
 
     @property
     def content_count(self):
-        return len((
+        return len(
             # we always want to call SMSMessageTemplate.__str__ regardless of subclass, to avoid any html formatting
             SMSMessageTemplate.__str__(self)
             if self._values
             else sms_encode(add_prefix(self.content.strip(), self.prefix))
```

Comment:
Out of interest, this appears to be formatting and tidying up the content of the message in two different ways here, based on whether there is `self._values`. Could this behaviour be a contender to move into `SMSMessageTemplate.__str__` itself? It seems like it would be nice to have all message formatting be done in one place. By the way, not suggesting you change that here (if you think it seems sensible that is) - just checking how well I've understood the code

Reply:
it doesn't quite work, unfortunately, because for placeholders that aren't filled it returns some HTML so that when they're rendered we highlight them in yellow (as seen when you send a one off message through the admin interface)
So in our tests, they fail because the count would be of `<span class='placeholder'>((placeholder))</span>` instead of `((placeholder))`. Perhaps the code could be refactored slightly but i'm a bit reluctant to prod this particular snake pit too much 😅

Reply:
Got ya!